src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "moviesessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinTable(name = "moviesessions_movies",
+            joinColumns = @JoinColumn(name = "moviesession_id"),
+            inverseJoinColumns = @JoinColumn(name = "movie_id"))
+    private Movie movie;
+    @ManyToOne
+    @JoinTable(name = "moviesessions_cinemahalls",
+            joinColumns = @JoinColumn(name = "moviesession_id"),
+            inverseJoinColumns = @JoinColumn(name = "cinemahall_id"))

```

Comment:
let's create only one column instead of a separate tables, it's a better approach